community/kernel/src/main/java/org/neo4j/kernel/impl/storemigration/participant/StoreMigrator.java

```diff
@@ -177,6 +182,15 @@ public void migrate( File storeDir, File migrationDir, MigrationProgressMonitor.
         // must be a proper translation happening while doing so.
     }
 
+    private void migrateWithDirectMigration( File storeDir, File migrationDir,
+            RecordFormats oldFormat, RecordFormats newFormat ) throws IOException
+    {
+        DirectRecordStoreMigrator migrator = new DirectRecordStoreMigrator( pageCache, fileSystem, config );
+        //Not interested in migrating MetaData store.
+        StoreType[] stores = ArrayUtil.filter( StoreType.values(), type -> type != StoreType.META_DATA );
+        migrator.migrate( storeDir, oldFormat, migrationDir, newFormat, stores, new StoreType[0] );
```

Comment:
last param can be represented ad vararg so you can avoid useless new StoreType[0]